src/NuGet.Shared/Entities/PackageFeed.cs

```diff
@@ -46,6 +46,12 @@ private PackageFeed(PackageSource source)
 
 			logMessage.AppendLine(versions.Length > 0 ? $"Found {versions.Length} versions" : "No versions found");
 
+			if(IsPrivate && author.HasValue())
+			{
+				logMessage.AppendLine($"Ignoring author filter for private feeds");
+				author = null;
```

Comment:
better to just add `!IsPrivate &&` to the next `if`

Reply:
I wanted to emphasize the fact that we're ignoring it with a message, but I guess I could just do that yes!

Reply:
Done!